mods/ra/maps/allies-01/allies01.lua

```diff
@@ -123,6 +123,10 @@ CiviliansKilled = function()
 	collateralDamage = true
 end
 
+LostMate = function()
+	Civilian2.Panic()
```

Comment:
This needs an `IsDead` check. (Except if this is just a testcase?)

Reply:
Yes, in theory the player could force fire on the 2nd civilian. Added the check. This is a test case with potential to stay I guess.